src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -0,0 +1,2085 @@
+
+When it comes accessing Enrollment data, it is important to have access to the Tracked Entity first. Access to a Tracked Entity is controlled through sharing setting of Program, Tracked Entity Type and Tracked Entity Attribute. Once Enrollment is accessed it is possible to access Event data, again depending on Program Stage and Data element sharing setting.
+
+Another important point to consider is how to map out access to different Program Stages of a Program. Sometimes we could be in a situation where we need grant access to a specific stage – for example “Lab Result” – to specific group of users (Lab Technicians). In this situation, we can provide data write access to "Lab Result" stage, probably data read to one or more stages just in case we want Lab Technicians to read other medical results or no access if we think it not necessary for the Lab Technicians to see data other than lab related.
+
+In summary, DHIS2 has fine grained sharing setting that we can use to implement access control mechanisms both at data and metadata level. These sharing settings can be applied directly at user level or user group level. How exactly to apply a sharing setting depends on the usecase at hand.
+
+### Organisation Unit Scopes
+
+<!--DHIS2-SECTION-ID:webapi_nti_ou_scope-->
+
+Organisation units are one of the most fundamental objects in DHIS2. They define a universe under which a user is allowed to record and/or read data. There are three types of organisation units that can be assigned to a user. These are data capture, data view and tracker search. As the name implies, these organisation units define a scope under which a user is allowed to conduct the respective operations.
+
+However, to further fine tune the scope, DHIS2 tracker introduces a concept that we call **OrganisationUnitSelectionMode**. Such a mode is often used at the time exporting tracker objects. For example, given a user has a particular tracker search scope does it mean that we have to use this scope every time a user tries to search for a tracker, enrollment or event object? Or is the user interested to limit the searching just to the selected org unit, or the entire capture org unit scope and so on. 
+
+Users can do the fine-tuning by passing a specific value of ouMode in their API request:
+
+*api/tracker?orgUnit=UID&ouMode=specific_organisation_unit_selection_mode*
```

Comment:
/api/tracker doesn't support GET, should we change the example to `api/tracker/trackedEntities` ?